WebUI/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "WebUI",
```

Comment:
Should probably put "OrbitWebUI" or something

Reply:
good point. I changed it to `orbit-web-ui` since camel case is not encouraged/wanted by npm. Doesn't really matter since we won't upload that package, but still. Thanks!